api/client/client_test.go

```diff
@@ -136,6 +136,83 @@ func TestClient_NewCollection(t *testing.T) {
 	})
 }
 
+func TestClient_UpdateCollection(t *testing.T) {
+	t.Parallel()
+	client, done := setup(t)
+	defer done()
+
+	id := thread.NewIDV1(thread.Raw, 32)
+	err := client.NewDB(context.Background(), id)
+	checkErr(t, err)
+	err = client.NewCollection(context.Background(), id, db.CollectionConfig{Name: collectionName, Schema: util.SchemaFromSchemaString(schema)})
+	checkErr(t, err)
+
+	t.Run("test update collection", func(t *testing.T) {
+		err = client.UpdateCollection(context.Background(), id, db.CollectionConfig{
+			Name:   collectionName,
```

Comment:
Nice, so looks like anything in collectionconfig is fair game for updating. Super.